include/roboteam_ai/stp/computations/PositionScoring.h

```diff
@@ -19,7 +19,7 @@ class PositionScoring {
      * @param scores ref to struct linked to that pos
      * @return score value
      */
-    static double determineOpenScore(Vector2 &point, const world::World *world, gen::PositionScores &scores);
+    static double determineOpenScore(Vector2 &point, const rtt::world::Field &field, const world::World *world, gen::PositionScores &scores);
```

Comment:
Why add field as a parameter when it can also be obtained through world?

Reply:
Speed. Passing it to the function as a separate argument speeds up the whole function by a factor of around 30 compared to extracting field from world on each call.